src/parser/jobs/rdm/modules/Procs.tsx

```diff
@@ -0,0 +1,192 @@
+			Try not to use <ActionLink {...this.data.actions.VERSTONE} /> while the boss is invulnerable.
+		</Trans>
+	}
+
+	private getInvulnProcWhy(invulnFire: number, invulnStone: number) {
+		if (invulnFire > 0 && invulnStone > 0) {
+			return <Trans id="rdm.procs.suggestions.invuln.why">
+				You used <Plural value={invulnFire} one="# Verfire proc" other="# Verfire procs" />, and <Plural value={invulnStone} one="# Verstone proc" other="# Verstone procs" /> on an invulnerable boss.
+			</Trans>
+		}
+		if (invulnFire > 0) {
+			return <Trans id="rdm.procs.suggestions.invuln-fire.why">
+				You used <Plural value={invulnFire} one="# Verfire proc" other="# Verfire procs" /> on an invulnerable boss.
+			</Trans>
+		}
+		return <Trans id="rdm.procs.suggestions.invuln-stone.why">
+			You used <Plural value={invulnStone} one="# Verstone proc" other="# Verstone procs" /> on an invulnerable boss.
+		</Trans>
+	}
+
+	protected addJobSpecificSuggestions() {
+		const missedFire = this.getDropCountForStatus(this.data.statuses.VERFIRE_READY.id)
+		const invulnFire = this.getInvulnCountForStatus(this.data.statuses.VERFIRE_READY.id)
+		const overWrittenFire = this.getOverwriteCountForStatus(this.data.statuses.VERFIRE_READY.id)
+		const missedStone = this.getDropCountForStatus(this.data.statuses.VERSTONE_READY.id)
+		const invulnStone = this.getInvulnCountForStatus(this.data.statuses.VERSTONE_READY.id)
+		const overWrittenStone = this.getOverwriteCountForStatus(this.data.statuses.VERSTONE_READY.id)
+
+		//Icons always default to the White Mana spell if black/jolt spells don't have more bad items.
+		//Fire/Stone are identical
+		this.suggestions.add(new TieredSuggestion({
+			icon: missedFire > missedStone ? this.data.actions.VERFIRE.icon : this.data.actions.VERSTONE.icon,
+			content: this.getMissedProcContent(missedFire, missedStone),
+			tiers: SEVERITY_MISSED_PROCS,
+			value: missedFire + missedStone,
+			why: this.getMissedProcWhy(missedFire, missedStone),
+		}))
+
+		this.suggestions.add(new TieredSuggestion({
+			icon: overWrittenFire > overWrittenStone ? this.data.actions.VERFIRE.icon : this.data.actions.VERSTONE.icon,
+			content: this.getOverwrittenProcContent(overWrittenFire, overWrittenStone),
+			tiers: SEVERITY_OVERWRITTEN_PROCS,
+			value: overWrittenFire + overWrittenStone,
+			why: this.getOverwrittenProcWhy(overWrittenFire, overWrittenStone),
+		}))
+
+		this.suggestions.add(new TieredSuggestion({
+			icon: invulnFire > invulnStone ? this.data.actions.VERFIRE.icon : this.data.actions.VERSTONE.icon,
+			content: this.getInvulnProcContent(invulnFire, invulnStone),
+			tiers: SEVERITY_INVULN_PROCS,
+			value: invulnFire + invulnStone,
+			why: this.getInvulnProcWhy(invulnFire, invulnStone),
+		}))
+	}
+
+	output() {
+		const allInvulns = this.getUsagesForStatus(this.data.statuses.VERFIRE_READY.id).concat(this.getInvulnsForStatus(this.data.statuses.VERSTONE_READY.id)).sort((a, b) => a.timestamp - b.timestamp)
```

Comment:
I suspect this should be getInvulnsForStatus instead, given the sheer number of invuln entries I'm seeing on random RDM logs I pull up. 🤔 

```suggestion
		const allInvulns = this.getInvulnsForStatus(this.data.statuses.VERFIRE_READY.id).concat(this.getInvulnsForStatus(this.data.statuses.VERSTONE_READY.id)).sort((a, b) => a.timestamp - b.timestamp)
```

Reply:
🤦